src/components/pages/course-page/CoursePage.tsx

```diff
@@ -0,0 +1,83 @@
+import React, { useState } from 'react';
+import { useQuery } from 'react-query';
+import { Alert, AlertTitle, CircularProgress } from '@mui/material';
+import { useRouter } from 'next/router';
+
+import { CoursersAPI } from '@/api/CoursesApi/CoursersAPI';
+import ListCourse from '@/components/common/list-course';
+import MediaContentCourse from '@/components/common/media-content-course/MediaContentCourse';
+import Navbar from '@/components/common/navbar';
+
+import styles from './CoursePage.module.scss';
+
+const CoursePage = () => {
+  const [selectedLesson, setSelectedLesson] = useState(0);
+  const router = useRouter();
+  const id = router.query.courseid as string;
+  const { isLoading, isError, data } = useQuery(
+    [['courses'], id],
+    () => CoursersAPI.getCourse(id),
+    { retry: false, refetchOnWindowFocus: false },
+  );
+  return (
+    <div className={styles['page-container']}>
+      <Navbar text={data?.title} />
+      {isLoading && <CircularProgress />}
+      {isError ? (
+        <Alert variant="filled" severity="error" className={'alert-error'}>
+          Finding problems with uploading data - Check later!
+        </Alert>
+      ) : (
+        !isLoading && (
```

Comment:
Difficult to read so many conditions inside HTML structure. Move it to the top, and write something like:

if (isLoading) return <Progress/>
if (isError) return ...

return *regular component*